README.md

```diff
@@ -6,18 +6,22 @@ Table of Contents
 * [Overview](#Overview)
 * [Setup](#Setup)
 * [How to Run Tests](#How-to-Run-Tests)
-* [Test Files](#Test-Files)
+* [TPCH Tests](#TPCH-Tests)
   * [Test Case Args](#Test-Case-Args)
   * [Substrait Plans](#Substrait-Plans)
   * [SQL Queries](#SQL-Queries)
-* [How to Add New Consumers](#How-to-Add-New-Consumers)
+* [Substrait Function Tests](#Substrait-Function-Tests)
+  * [Test Case Args](#Test-Case-Args)
+  * [SQL Queries](#SQL-Queries)
+  * [Ibis Expressions](#Ibis-Expressions)
+* [How to Add Producers](#How-to-Add-Producers)
+* [How to Add Consumers](#How-to-Add-Consumers)
+
 
 # Overview
 This testing repository provides instructions on how to add and run substrait consumer 
-integration tests.  These tests take in parquet files, a SQL query, and a json-formatted substrait 
-query plan as input.  The tests will run the SQL query on DuckDB, and the substrait plan on the 
-specified consumer, and then compare the two results. The test data is created using DuckDB at the 
-start of the test class using the `prepare_tpch_parquet_data` fixture, which is located in 
+integration tests.  The tests are organized into two categories; tpch tests and substrait function tests.
```

Comment:
I don't think we need to fix this right now but I'm not sure these are just "consumer integration" tests any longer.  I think you're doing a fair amount of producer integration testing as well.  Though given the repo name is `consumer-testing` I think a change here would be a future PR.